### PR TITLE
update bankrate theme

### PR DESCRIPTION
**Pull request by phelma**
# Description

update bankrate theme

# Checklist

Pull request contains:

- [ ] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [ ] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.
- [ ] If component change, version updated

### CODE REVIEW

File: src/themes/bankrate/theme.json [hunk 1]
```diff
@@ -219,26 +221,12 @@
...
-        "color": "grey-80",
-        "::before": {
-          "content": "\" - \" counter(li)",
-          "left": "-2em",
-          "width": "2.5em"
-        }
-      }
-    }
-  },
-
```
Comment (domtronn) [inline src/themes/bankrate/theme.json:237]: This needs adding back in and the deprecated top level list being removed.

Comment (phelma) [reply src/themes/bankrate/theme.json]: like this? https://github.com/uswitch/trustyle/pull/596/commits/acae2369566984e261b8ecdf9200f43d65e74c7c

Comment (domtronn) [reply src/themes/bankrate/theme.json]: Why remove the before content as well? Do we not want that for the theme?

Comment (phelma) [reply src/themes/bankrate/theme.json]: I was just copying across what we had donne in the glaceon repo, but I think that the :before stuff lives in the component for all brands

**Pull request merged**